tests/test-infra/azure-aks.bicep

```diff
@@ -26,6 +29,9 @@ param linuxVMSize string = 'Standard_DS2_v2'
 @description('VM size to use for Windows nodes, if enabled')
 param windowsVMSize string = 'Standard_DS3_v2'
 
+@description('VM size to use for Arm64 nodes if enabled')
+param armVMSize string = 'Standard_D4pds_v5'
```

Comment:
This seems a pretty beefy VM. On Linux we use a DS2 which has 2 cores only. This one has 4

Reply:
I tried using a smaller one, but I received an VMCannotFitEphemeralOSDisk when trying to deploy.